app/namespace.go

```diff
@@ -65,34 +83,43 @@ func (c *NamespaceClient) getNamespace(namespace string) (*ns.Namespace, error)
 	return res.Namespace, nil
 }
 
-func (c *NamespaceClient) updateNamespace(
-	namespace string,
-	requestID string,
-	resourceVersion string,
-	spec *ns.NamespaceSpec,
-) error {
+func (c *NamespaceClient) updateNamespace(ctx *cli.Context, n *ns.Namespace) error {
+	if n.State != ns.STATE_ACTIVE {
```

Comment:
should we rely on server for this kind of validation instead of putting this here?

Reply:
You are right, we should...
Removed the check here.
Currently we do the check in the entity, but I will add the check on the service itself.